nx.json

```
@@ -0,0 +1,11 @@
+{
+  "extends": "nx/presets/npm.json",
```

Let's add the schema as well for autocomplete/lint in vscode
```suggestion
  "$schema": "./node_modules/nx/schemas/nx-schema.json",
  "extends": "nx/presets/npm.json",
```